### PR TITLE
Add Syncthing monitoring 

### PR DESCRIPTION
**Pull request by edupr91**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -688,6 +688,32 @@ Add the following on the client manifest:
    nagios::check::service { 'foo_service': }
 ```
 
+## Syncthing
+
+Check the Syncthing status via de API
+You'll need to they API key from your node.
+
+```yaml
+nagios::check::mongodb::api_key: 'l0r2m1psumd0l0rs1t4m3tC0ns3ct3tur'
```
Comment (thias) [inline README.md:697]: mongodb, really? :sweat_smile:

Comment (edupr91) [reply README.md:697]: I'm not proud of myself right now...

**Comment by edupr91** [comment]
@thias, can you please accept this pull request, I've fixed the  README :-)

**Comment by edupr91** [comment]
Fixed conflicts

**Comment by edupr91** [comment]
Fixed again ^^'

**Pull request merged**